test/flop_phoenix_test.exs

```diff
@@ -1200,7 +1219,10 @@ defmodule Flop.PhoenixTest do
 
       # reverse
       html =
-        render_cursor_pagination(meta: build(:meta_with_cursors), reverse: true)
+        render_cursor_pagination(
+          meta: build(:meta_with_cursors),
+          reverse: true
+        )
```

Comment:
This line and several others failed the Code Quality CI check due to line-length, despite my branch not changing them and the build currently being green on `main` 🤷🏻‍♂️

Reply:
🤷‍♀️